setup.py

```diff
@@ -8,7 +8,7 @@
 install_requires = [
     "Jinja2>=2.7.3",
     "boto>=2.36.0",
-    "boto3>=1.6.16",
+    "boto3==1.7.84",
```

Comment:
it will force the tested app installs `boto3@1.7.84` which may be unexpected.

What about `"boto3>=1.6.16,<1.8",`?

Reply:
What about just `<1.8`?

Reply:
The 1.6.16 minimum was set in https://github.com/spulec/moto/pull/1535 so Moto has some dependencies on at least that version. @shawnzhu's suggestion looks good to me until this botocore issue can be resolved.